website/docs/registry/providers/docs.mdx

```diff
@@ -11,6 +11,8 @@ The [Terraform Registry][terraform-registry] displays documentation for the prov
 
 -> In order to test how documents will render in the Terraform Registry, you can use the [Terraform Registry Doc Preview Tool](https://registry.terraform.io/tools/doc-preview).
 
+-> Please note that this currently isn't supported in the private registry of [Terraform Cloud/Terraform Enterprise](https://developer.hashicorp.com/terraform/cloud-docs/registry/publish-providers#release-files)
```

Comment:
Is the "this" here meant to be "provider documentation" or the preview tool? If so, I'd suggest moving this up above the other callout to raise it's importance and specifying the "this" 👍

Reply:
I made some changes to the pull request. Is this better?

Reply:
I've requested a review from the documentation team, I do think the prose is at least a little clearer now for readers. 👍